dioptas/model/ImgModel.py

```diff
@@ -250,7 +250,7 @@ def load_karabo(self, filename):
         except IOError:
             return None
 
-        return {"img_data": karabo_file.get_image(3),
+        return {"img_data": karabo_file.get_image(0),
```

Comment:
I changed this as the docstring mentioned that it should return the first image in file.